packages/react/src/components/dialog/Dialog.module.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../styles/common';
+@value medium-up from "../../styles/breakpoints.css";
+
+//for document body element
+.dialogVisibleBody {
+  height: 100vh !important;
+  overflow-y: hidden !important;
+}
+
+.dialogContainer {
+  --accent-line-color: var(--color-bus);
+  --overlay-color: rgba(0, 0, 0, 0.3);
+  --width: 558px;
+
+  align-items: center;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 800;
+  overflow: auto;
+
+  .dialogBackdrop {
+    background: var(--overlay-color);
+  }
+
+  .dialog {
+    border-top: 10px solid var(--accent-line-color);
+    width: var(--width);
+  }
+}
+
+.dialogBackdrop {
+  bottom: 0;
+  height: 100%;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 100%;
+}
+
+.dialog {
+  background: white;
+  box-sizing: border-box;
+  margin: auto;
+  max-height: max-content;
```

Comment:
Just out of curiosity, what does this mean? :)

Reply:
https://ishadeed.com/article/intrinsic-sizing-in-css/#max-content
I will remove this, it probably has no effect.